scvi/dataset/dataset.py

```diff
@@ -241,28 +241,29 @@ def populate_from_datasets(
         For cell-wise attributes, either we "concatenate" or add an "offset" corresponding
         to the number of already existing categories.
 
-        :param gene_datasets: a sequence of ``GeneExpressionDataset`` objects.
-        :param on: attribute to select gene intersection
-        :param sharing_instructions_dict: Instructions on how to share cell-wise attributes between datasets.
-            Keys are the attribute name and values are either:
-                * "offset": to add an offset corresponding to the number of categories already existing.
-                e.g for batch_indices, if the first dataset has batches 0 and 1,
-                in the merged dataset, the second dataset's batch indices will start at 2.
-                * "concatenate": concatenate the attribute, no changes applied.
+        :param gene_datasets_list: ``GeneExpressionDataset`` objects to be merged.
+        :param shared_labels: whether to share labels through ``cell_types`` mapping or not.
+        :param mapping_reference_for_sharing: Instructions on how to share cell-wise attributes between datasets.
```

Comment:
I don't really understand the docstring for `mapping_reference_for_sharing`.
You don't have `offset` or `concatenate` anymore?

Reply:
Ok I got it, you should indicate what happens if the mapping is not provided, here we only have "If provided the mapping is merged ..." . Also i don't understand what is the meaning of "merged" here.

Reply:
no i changed that. edited the docstring. tell me if its clearer

Reply:
I still don't understand the usage of "merged" here `If provided the mapping is merged across all datasets`. The docstring is supposed to explain how the merging is done and to use "the mapping is merged" confuses me a bit.
For instance if I have: 
#### Dataset 1
`cell_types = ["T cells", "B cells"]` and `labels = [0,1,0,0,1]`
#### Dataset 2
`cell_types = ["B cells - 1", "B cells - 2", "T cells"]` and `labels = [1,2,0,0,1]`
####
Then what merging means? To have `labels = [0,1,0,0,1] + [1,2,0,0,1]`? And what about `cell_types`

Thanks!

Reply:
In essence, you take a mapping e.g ``cell_types`` you merge all of those into a set of aggregated mapping ``["T cells", "B cells", "B cells - 1", "B cells - 2"]`` then remap each datasets attribute, e.g ``labels``, using the previous and aggregated mapping.